src/common/lib/client/realtimechannel.ts

```diff
@@ -596,11 +587,18 @@ class RealtimeChannel extends Channel {
   }
 
   onMessage(message: ProtocolMessage): void {
+    const isNotControlMsg = message.action === actions.MESSAGE || message.action === actions.PRESENCE;
+    if (isNotControlMsg || message.action === actions.ATTACHED) {
+      const suppressed = this.setChannelSerial(message);
```

Comment:
this always returns `false`, do we even need `suppress`?

Reply:
nope - that was added before we decided duplicates we're not possible with the new upgrade scheme - thanks for spotting will remove